CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contribution guidelines
+
+### Who are we?
+
+*Maintainers* (in alphabetical order: Viacheslav Borovitskiy, Vincent Dutordoir, Michael Hutchinson, Noemie Jaquier,  Peter Mostowsky, Aditya Ravuri, Leonel Rozo, Alexander Terenin) steer the project, keep the community thriving, and manage contributions.
```

Comment:
Maybe we should discuss the list in Slack, check if everybody wants to be in this list and is ready to carry some responsibility associated to that.